src/gitlab/functions/add-badges.ts

```diff
@@ -54,10 +54,30 @@ export function addBadges() {
         }
         badges += `</tr>`;
 
+        badges += `<tr>`;
```

Comment:
Please only add new line if there is something to display, most repositories don't have nightly pipelines and this will extend the default UI

Reply:
I did it for the tags as well (as some repositories might not have tags either)